app/templates/base.html

```diff
@@ -20,8 +21,14 @@
 {% block header %}
   {{ govukHeader({
     'homepageUrl': url_for('main.download'),
-    'serviceName': config['SERVICE_NAME'],
-    'serviceUrl': url_for('main.download')
+    'serviceName': 'Find monitoring data',
+    'serviceUrl': url_for('main.download'),
+    'navigation': [
+    {
+      'href': config['AUTHENTICATOR_HOST'] + "/sso/logout",
```

Comment:
do we want to log out of SSO like this (also ending our session with authenticator) - or do we just want to end our authenticator session by going to `sessions/sign-out?return_app=post-award-frontend`. This would mean that if the user clicks log-in again they log straight back in with their SSO still being active and wouldn't have to enter their SSO details again. I don't know what the answer to this is but it's a different option...

otherwise this all looks good 👍